### DISCUSSION TITLE
Find Public Folders with Rules

### DISCUSSION THREAD

**Discussion by pipyoung** [microsoft/mfcmapi#508] [Q&A]
Using MFCMapi I can see the PR_HAS_RULES property. I'd like to use MRMapi to search the public folder tree to see which folders have rules set on the,. Is this possible?
regards
Phil

**Comment by stephenegriffin** [comment]
Hrmm - -childfolders will list childfolders but I don't think I added a way to dump properties on them all at once

**Reply by pipyoung** [reply]
Thanks Stephen, that's a shame but at least I can see a True or False for each folder ..guess I'll have to trawl through the Hierarchy!
